Hello @dbv-rl you PR is failing because of the following:

```
--> Executing Yamllint on files found in /home/travis/build/victorock/ansible-role-tower-config/...
    /home/travis/build/victorock/ansible-role-tower-config/tasks/config/organization/inventory.yml
      22:4      warning  missing starting space in comment  (comments)
      22:3      warning  comment not indented like content  (comments-indentation)
      23:4      warning  missing starting space in comment  (comments)
```

--- a/tasks/config/organization/inventory.yml
+++ b/tasks/config/organization/inventory.yml
@@ -19,5 +19,3 @@
   loop_control:
     loop_var: tower_config_organization_inventory_source
     label: "{{ tower_config_organization_inventory_source.name }}"
-  #async: 15
-  #poll: 1

